bridge-cli/src/omni_connector_command.rs

```diff
@@ -1,35 +1,53 @@
-use crate::{combined_config, CliConfig, Network};
+use std::str::FromStr;
+
 use clap::Subcommand;
+
 use ethers_core::types::TxHash;
+use evm_bridge_client::EvmBridgeClientBuilder;
+use near_bridge_client::NearBridgeClientBuilder;
 use near_primitives::{hash::CryptoHash, types::AccountId};
-use omni_connector::{OmniConnector, OmniConnectorBuilder};
-use omni_types::{
-    locker_args::{BindTokenArgs, FinTransferArgs},
-    ChainKind, Fee,
+use omni_connector::{
+    BindTokenArgs, DeployTokenArgs, FinTransferArgs, InitTransferArgs, LogMetadataArgs,
+    OmniConnector, OmniConnectorBuilder,
 };
-use std::str::FromStr;
+use omni_types::{ChainKind, Fee};
+use solana_bridge_client::SolanaBridgeClientBuilder;
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::signature::Keypair;
+
+use crate::{combined_config, CliConfig, Network};
 
 #[derive(Subcommand, Debug)]
 pub enum OmniConnectorSubCommand {
-    LogMetadata {
+    NearLogMetadata {
         #[clap(short, long)]
         token: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
-    StorageDeposit {
+    NearDeployToken {
+        #[clap(short, long)]
+        source_chain_id: u8,
+        #[clap(short, long)]
+        vaa: String,
```

Comment:
It would be great to have the option to use a non-Wormhole prover here. This could be a separate task in a new PR. Could you create an issue for this?

Reply:
https://github.com/Near-One/bridge-sdk-rs/issues/55